### ISSUE TITLE
Consider only syncing live poll states

### ISSUE THREAD

**Issue by nathansherburn** [SLC3/MARS#151]
This simplifies the code and probably makes more sense.

You don't want your audience watching as you look through your trash, open your next set of questions etc.

**Comment by aj1331** [comment]
It's a lot easier for people to understand "the views are synced" than "these specific views are synced". The way it's written at the moment, it uses the `$stateChangeSuccess` event to do the synchronisation. It doesn't simplify anything to only sync certain views. I would like to replace the call to `viewSyncService.ignoreStates()` with an attribute on the state itself though eg.
```
$stateProvider.state('myPolls.trash', {
  url: '/trash',
  templateUrl: 'app/my-polls/trash/trash.html',
  controller: 'TrashController as vm',
  shouldSync: false
})
```

**Comment by nathansherburn** [comment]
Good point. I'll remove this from the 1.0.0 milestone but the should sync state variables are a nice idea.

On a related note, when I moved all the dialogues into the controllers (and out of the states) there was no longer a need for any ignore states. It's likely we'll have some in the future though so I'll keep this open for the idea above.
